templates/http-rust/metadata/spin-template.toml

```diff
@@ -2,6 +2,12 @@ manifest_version = "1"
 id = "http-rust"
 description = "HTTP request handler using Rust"
 
+[add_component]
+skip_files = ["spin.toml"]
+skip_parameters = ["http-base"]
+[add_component.snippets]
```

Comment:
I did wonder about that!  It might need a bit of extra magic to make the paths work but that should just be a Liquid filter.  I am a little concerned about that being a breaking change, though I think technically the new section is a breaking change anyway (our TOML reader is set to fail on unrecognised elements).  I will ponder this more - thanks!